### PR TITLE
DTSCCI-269 Remove code smell

### PR DESCRIPTION
**Pull request by tiagreaney**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DTSCCI-269

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by rishikrsharma — DISMISSED** [review]
LGTM

**Review by velimd — DISMISSED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/callback/CallbackHandler.java [hunk 1]
```diff
@@ -40,6 +40,7 @@ protected String callbackKey(CallbackVersion version, CallbackType type, String
         return String.format("%s%s%s", formattedVersion, type.getValue(), formattedPageId);
     }
 
+    @SuppressWarnings("java:S1172")
     public String camundaActivityId(CallbackParams callbackParams) {
```
Comment (velimd) [inline src/main/java/uk/gov/hmcts/reform/civil/callback/CallbackHandler.java:44]: in this case, maybe the warning is correct as callbackParams doesnt seem to be used.

File: src/main/java/uk/gov/hmcts/reform/civil/callback/DashboardCallbackHandler.java [hunk 2]
```diff
@@ -31,6 +31,7 @@ protected Map<String, Callback> callbacks() {
 
     protected abstract String getScenario(CaseData caseData);
 
+    @SuppressWarnings("java:S1172")
     protected String getExtraScenario(CaseData caseData) {
```
Comment (velimd) [inline src/main/java/uk/gov/hmcts/reform/civil/callback/DashboardCallbackHandler.java:35]: same here for the warning.

File: src/main/java/uk/gov/hmcts/reform/civil/callback/DashboardCallbackHandler.java [hunk 3]
```diff
@@ -45,6 +46,7 @@ protected boolean shouldRecordScenario(CaseData caseData) {
         return true;
     }
 
+    @SuppressWarnings("java:S1172")
     protected boolean shouldRecordExtraScenario(CaseData caseData) {
```
Comment (velimd) [inline src/main/java/uk/gov/hmcts/reform/civil/callback/DashboardCallbackHandler.java:50]: here as well

File: src/main/java/uk/gov/hmcts/reform/civil/callback/DashboardWithParamsCallbackHandler.java [hunk 4]
```diff
@@ -31,6 +31,7 @@ protected Map<String, Callback> callbacks() {
 
     protected abstract String getScenario(CaseData caseData, CallbackParams callbackParams);
 
+    @SuppressWarnings("java:S1172")
     protected boolean shouldRecordScenario(CaseData caseData) {
```
Comment (velimd) [inline src/main/java/uk/gov/hmcts/reform/civil/callback/DashboardWithParamsCallbackHandler.java:35]: as above

File: src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/docmosis/GenerateClaimFormForSpecCallbackHandler.java [hunk 5]
```diff
@@ -140,6 +141,7 @@ private CallbackResponse generateClaimFormForSpec(CallbackParams callbackParams)
 
     }
 
+    @SuppressWarnings("java:S1172")
     private List<DocumentMetaData> fetchDocumentsFromCaseData(CaseData caseData, CaseDocument caseDocument,
                                   CaseData.CaseDataBuilder<?, ?> caseDataBuilder, CallbackParams callbackParams) {
```
Comment (velimd) [inline src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/docmosis/GenerateClaimFormForSpecCallbackHandler.java:146]: doesnt seem like these 2 parameters are used.

File: src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/ClaimIssuedPaymentSuccessfulNotificationHandler.java [hunk 6]
```diff
@@ -61,6 +61,7 @@ private CallbackResponse notifyClaimantRejectRepayment(CallbackParams callbackPa
         return AboutToStartOrSubmitCallbackResponse.builder().build();
     }
 
+    @SuppressWarnings("java:S1172")
     private String addTemplate(CaseData caseData) {
```
Comment (velimd) [inline src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/ClaimIssuedPaymentSuccessfulNotificationHandler.java:65]: .

**Review by velimd — COMMENTED** [review]
I think some of the warnings are valid and might need cleaning instead of the suppressWarning annotation.

File: src/main/java/uk/gov/hmcts/reform/civil/callback/CallbackHandler.java (hunk 1 above)
Comment (ruban72) [reply src/main/java/uk/gov/hmcts/reform/civil/callback/CallbackHandler.java:44]: I think this methods are intended to be overridden which is why they the args are not used here

**Review by ruban72 — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/callback/DashboardWithParamsCallbackHandler.java (hunk 4 above)
Comment (ruban72) [reply src/main/java/uk/gov/hmcts/reform/civil/callback/DashboardWithParamsCallbackHandler.java:35]: Same as above the method has overrides which actually use the method arg

**Review by rishikrsharma — APPROVED** [review]

**Pull request merged**